Makefile

```diff
@@ -58,6 +59,24 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
+ifeq ($(ARCH),s390x)
+K8S_VERSION=v1.25.5
+ETCD_VERSION=v3.5.5
+ETCD_BASE_NAME=etcd-$(ETCD_VERSION)-linux-$(ARCH)
+KUBEBUILDER_PATH=$${PWD}/kubebuilder-tools/bin
```

Comment:
Just to double-check, is this supposed to have `$${PWD}`?

Reply:
yep, at the Jenkins `$WORKSPACE`. But we could go with `mktemp -d`. I will follow your suggestion.

Reply:
Choi has let me know that `$${PWD}` is a special form of PWD for running in a `Makefile`, which I didn't know, so happy with this now I've learnt something!